### PR TITLE
Click to zoom

### PR DESCRIPTION
**Pull request by spupett**
I have only looked at the click to zoom functionality as it relates to the inner window zoom.  I haven't tested it for the windows outside, or if it works for the lens shaped viewer.

### CODE REVIEW

File: jquery.elevatezoom.js [hunk 1]
```diff
@@ -51,8 +51,12 @@ if ( typeof Object.create !== 'function' ) {
 					self.options.lensOpacity =  "1" //opacity of the lens
 				}
 				//INNER OVERRIDE SETTINGS
-				if(self.options.zoomType == "inner") {self.options.showLens = false;}
+				if(self.options.zoomType == "inner") {
```
Comment (skoberlink) [inline jquery.elevatezoom.js]: These lines need to be aligned correctly.

File: jquery.elevatezoom.js [hunk 2]
```diff
@@ -401,6 +400,40 @@ if ( typeof Object.create !== 'function' ) {
...
+                       self.options.zoomEnabled = !self.options.zoomEnabled;
+                       if(!self.options.zoomEnabled) {
+                           self.showHideLens('hide');
+                           self.showHideWindow('hide');
+                           self.zoomContainer.trigger('mouseleave');
+                        }
+                       else {
+                           self.showHideLens('show');
+                           self.showHideWindow('show');
+                           self.zoomContainer.trigger('mousemove'); //If this isn't here, there lens and zoom window won't show up untill the mouse is moved
```
Comment (skoberlink) [inline jquery.elevatezoom.js]: Is there a more direct way to show the lens and zoom window?  What does the mousemove event trigger? Should it be refactored to extract the method?

File: jquery.elevatezoom.js [hunk 3]
```diff
@@ -550,7 +584,11 @@ if ( typeof Object.create !== 'function' ) {
 				if(type=="show"){
 					if(self.isWindowSet){
 						if(self.options.zoomType == "inner") {self.showHideWindow("show");}
-						if(self.options.zoomType == "window") {self.showHideWindow("show");}
+						if(self.options.zoomType == "window") {
+                            if(self.options.zoomEnabled) {
```
Comment (skoberlink) [inline jquery.elevatezoom.js]: alignment is off here as well

File: jquery.elevatezoom.js [hunk 4]
```diff
@@ -795,7 +835,13 @@ if ( typeof Object.create !== 'function' ) {
 						if(self.options.lensFadeIn){
 							self.zoomLens.stop(true, true, false).fadeIn(self.options.lensFadeIn);
 						}
-						else{self.zoomLens.show();}
+						else{
+                            if(self.options.zoomEnabled) {
```
Comment (skoberlink) [inline jquery.elevatezoom.js]: alignment is off

**Comment by spupett** [comment]
@skoberlink Cleaned the formatting up a bit.  Should be easier on the eyes now.

**Comment by skoberlink** [comment]
@spupett Ironically, there are so many additions and deletions from that conversion that I can't even see the diff now.  Did you address the last comment that I didn't delete?  Regarding extracting the mousemove event trigger?

**Comment by spupett** [comment]
@skoberlink the mousemove and mousedown events have been refactored into their own functions.

**Pull request merged**